nixos/modules/services/ttys/kmscon.nix

```
@@ -142,7 +142,7 @@ in
           )
         );
         render = optionals cfg.hwRender [
-          "drm"
+          #          "drm"
```

```suggestion
          # "drm"
```
this should get a comment why it is commented or be remove completely

---

ok